test/e2e/scenarios/multiple_clusters.go

```diff
@@ -0,0 +1,27 @@
+package scenarios
+
+import (
+	. "github.com/onsi/ginkgo"
+)
+
+var _ = PDescribe("[Dataplane] Namespace selector based Network Policies for ingress between clusters, selecting multiple remote clusters", func() {
+	PIt("Should allow communication between pods in selected namespace across multiple clusters", func() {
+		By("creating listener pod 1 with label 1 in cluster 1 in namespace 1")
```

Comment:
you need to create 2 listener pods in cluster 1

Because listener pods "exit" as soon as one of the connector pod has connected to them.